src/schema/search.ts

```diff
@@ -0,0 +1,66 @@
+import { GraphQLList, GraphQLEnumType, GraphQLString } from "graphql"
+import { assign } from "lodash"
+
+import { Searchable } from "./searchable"
+
+export const SearchIndex = new GraphQLEnumType({
+  name: "SearchIndexes",
+  values: {
+    ARTWORK: {
+      value: "Artwork",
+    },
+    ARTIST: {
+      value: "Artist",
+    },
+  },
+})
+
+export const SearchMode = new GraphQLEnumType({
+  name: "SearchMode",
+  values: {
+    AUTOSUGGEST: {
+      value: "AUTOSUGGEST",
+    },
+    SITE: {
+      value: "SITE",
+    },
+  },
+})
+
+export const searchArgs = {
+  query: {
+    type: GraphQLString,
+  },
+  indexes: {
+    type: new GraphQLList(SearchIndex),
+  },
+  mode: {
+    type: SearchMode,
+  },
```

Comment:
I assume these are optional as the upstream service will have sane defaults? (A final version of the PR should include such defaults in the descriptions or the args.)